test/e2e/lib/components/sidebar-component.js

```diff
@@ -38,99 +39,102 @@ export default class SidebarComponent extends AsyncBaseContainer {
 	}
 
 	async selectPeople() {
-		await this.expandDrawerItem( 'Manage' );
-		return await this._scrollToAndClickMenuItem( 'people' );
+		await this.expandDrawerItem( 'Users' );
+		return await this._scrollToAndClickMenuItem( 'All Users' );
 	}
 
 	async selectThemes() {
-		await this.expandDrawerItem( 'Design' );
-		return await driverHelper.clickWhenClickable(
-			this.driver,
-			By.css( '.menu-link-text[data-e2e-sidebar="Themes"]' )
-		); // TODO: data-tip-target target is missing
+		await this.expandDrawerItem( 'Appearance' );
+		return await this._scrollToAndClickMenuItem( 'Themes' );
+	}
+
+	async selectAllSitesThemes() {
+		return await this._scrollToAndClickMenuItem( 'Themes' );
 	}
 
 	async selectSiteEditor() {
-		await this.expandDrawerItem( 'Design' );
 		return await driverHelper.clickWhenClickable(
 			this.driver,
-			By.css( '.menu-link-text[data-e2e-sidebar="Site Editor (beta)"]' )
+			By.css( '.menu-link-text[data-e2e-sidebar*="Site Editor"]' )
 		);
 	}
 
 	async selectWPAdmin() {
-		return await this._scrollToAndClickMenuItem( 'wpadmin' );
+		//Wp-admin isn't in nav-unification. Workaround to get to wp-admin
+		await this.expandDrawerItem( 'Feedback' );
+
+		if ( driverManager.currentScreenSize() === 'mobile' ) {
+			return await this._scrollToAndClickMenuItem( 'Feedback' );
+		}
 	}
 
 	async customizeTheme() {
-		return await this._scrollToAndClickMenuItem( 'themes' );
+		await this.expandDrawerItem( 'Appearance' );
+		return await this._scrollToAndClickMenuItemByText( 'Customize' );
 	}
 
 	async selectPlans() {
 		await this.expandDrawerItem( 'Upgrades' );
-		return await this._scrollToAndClickMenuItem( 'plans' );
+		return await this._scrollToAndClickMenuItem( 'Plans' );
 	}
 
 	async selectDomains() {
 		await this.expandDrawerItem( 'Upgrades' );
-		return await this._scrollToAndClickMenuItem( 'domains' );
+		return await this._scrollToAndClickMenuItem( 'Domains' );
 	}
 
 	async selectMyHome() {
-		return await this._scrollToAndClickMenuItem( 'myhome' );
+		return await this._scrollToAndClickMenuItem( 'My Home' );
 	}
 
 	async selectStats() {
-		return await this._scrollToAndClickMenuItem( 'stats' );
+		return await this._scrollToAndClickMenuItem( 'Stats' );
 	}
 
 	async selectActivity() {
 		await this.expandDrawerItem( 'Jetpack' );
-		return await this._scrollToAndClickMenuItem( 'activity' );
+		return await this._scrollToAndClickMenuItemByText( 'Activity Log' );
 	}
 
 	async selectMarketing() {
 		await this.expandDrawerItem( 'Tools' );
-		return await this._scrollToAndClickMenuItem( 'marketing' );
+		return await this._scrollToAndClickMenuItem( 'Marketing' );
 	}
 
 	async selectViewThisSite() {
-		return await this._scrollToAndClickMenuItem( 'sitePreview' );
+		return await this._scrollToAndClickMenuItemByText( 'sitePreview' );
```

Comment:
Is there a pattern for using `_scrollToAndClickMenuItemByText` instead of `_scrollToAndClickMenuItem`

Reply:
I had to look in to this and it turned out I didn't need the new one. Removed `_scrollToAndClickMenuItemByText`